srcpkgs/gnuradio/template

```diff
@@ -1,32 +1,31 @@
 # Template file for 'gnuradio'
 pkgname=gnuradio
-version=3.7.13.5
-revision=2
+version=3.8.0.0
+revision=1
 build_style=cmake
-nocross="relies on volk, which is not complete for arm"
 conf_files="/etc/gnuradio/conf.d/*"
 configure_args="-DENABLE_INTERNAL_VOLK=OFF"
 pycompile_module="gnuradio grc_gnuradio pmt"
 hostmakedepends="pkg-config doxygen swig python-cheetah python-lxml
  python-numpy python-Sphinx"
 makedepends="SDL-devel boost-devel fftw-devel gsl-devel jack-devel libgsm-devel
- libusb-devel pygtk-devel volk-devel wxPython-devel uhd-devel python-devel"
+ libusb-devel pygtk-devel volk-devel wxPython-devel uhd-devel python-devel
+ log4cpp-devel gmpxx-devel mpir-devel"
 depends="pygtk python-cheetah python-lxml python-numpy"
 short_desc="Framework for software defined radio"
 maintainer="Andrew Benson <abenson+void@gmail.com>"
 license="GPL-3.0-or-later"
 homepage="https://www.gnuradio.org"
 distfiles="https://github.com/gnuradio/gnuradio/archive/v${version}.tar.gz"
-checksum=9afcd0e4499160bce17c8c5c100f461791ad26b4bcb967d956fb9567e74dd57f
+checksum=03fd01f0b27033befb91709b3d56f95e0869b5fb8bc71bec2662266a3f0dedf5
+nocross="relies on volk, which is not complete for arm"
 
 if [ "$CROSS_BUILD" ]; then
 	nocross="https://build.voidlinux.org/builders/armv7l_builder/builds/7596/steps/shell_3/logs/stdio"
 fi
 
 post_install() {
-	rm ${DESTDIR}/usr/share/gnuradio/examples/audio/dial_tone \
-		${DESTDIR}/usr/share/gnuradio/examples/fcd/fcd_nfm_rx \
```

Comment:
Because I don't want to have to hunt for every ELF example that's different in each release.

Reply:
You usually don't have to hunt down anything, `xbps-src` tells you which binary needs to be removed.

Completly removing the examples, just because of two binaries, is totally overkill